go.mod

```diff
@@ -4,6 +4,8 @@ go 1.18
 
 replace github.com/ThalesIgnite/crypto11 => github.com/aoscloud/crypto11 v1.0.3-0.20220217163524-ddd0ace39e6f
 
+replace github.com/aoscloud/aos_common => ../aos_common
```

Comment:
Fromy point of view it is incorrect commit message, because there is no any removed packages